CONTRIBUTING.md

```diff
@@ -55,131 +54,265 @@ Please follow these steps and note these guidelines to begin contributing:
 
+1. In order to add/update news items, fork this repository, clone it, create a new branch, and navigate to the `src/collections/news` directory.
+2. Create a copy of the [news template](https://github.com/layer5io/layer5/tree/master/src/collections/news/news-template/0000-00-00-news-title).
+3. Follow the instructions included in the news template and name the new file after the title of the news article.
+4. Entries will be listed in chronological order automatically.
 
 # Common Types of Site Contributions
 
 The following list of instructions pertains to commonplace site updates by contributors.
 
 ## Things to keep in mind
 
-  1. Use only the theme colors available at [themeStyles.js](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js) instead of external color codes.
-  2. Avoid adding `rel="noreferrer"` attribute to internal links on the site.
+1. Prioritize using only the theme colors and hexcodes that are already being used in [themeStyles.js](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js) instead of introducing new color codes to the file.
+2. Avoid adding `rel="noreferrer"` attribute to internal links on the site.
+
+## Adding dark mode for components
+
+The site currently has dark and light mode, the colors used in these two modes can be found in the dark and light theme objects in [themeStyles.js](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js).
+
+These theme objects use a property whose value automatically changes based on color mode without using a conditional.
+
+An example of a conditional:
+
+```
+theme === "dark" ? "white" : "black"
+```
+
+The preference is to not use conditionals for elements because they will flicker when reloading a page in dark mode.
+
+As currently constructed, the dark and light theme objects share similar property names that have different values.
+
+The property name will tell you how the value (color or hexcode) will change from dark mode to light mode. The first part is for dark mode, then "To", and the next part is for light mode.
+
+example #1: `whiteToBlack`
 
-## Adding dark theme for components
+"white" is the first part for dark mode
+"To"
+"Black" is the second part for light mode
 
-1. To differentiate between the colors of the dark and light theme, we have created a customization system in which you can define styles like color, position, display, etc using {props => props.theme.DarkTheme? darkThemeStyle: LightThemeStyle} 
-  
+In dark mode the value of `whiteToBlack` will be `white`
+In light mode the value of `whiteToBlack` will be `black`
 
-2. It is recommended to use pre-existing styles or colors from the theme. Avoid adding your CSS variables."
+example #2: `grey141414ToGreyF5F5F5`
 
+In dark mode this value will be `#141414` (a dark shade of grey)
+In light mode this value will be `#F5F5F5` (a light shade of grey).
+
+Colors in the property name give an indication of the color shade of a hexcode.
```

Comment:
Better:
The hexcode in the property name is the indication of the color shade being used in the 2 modes.